android/app/src/main/java/app/organicmaps/car/util/ThemeUtils.java

```diff
@@ -88,13 +88,14 @@ public static void setThemeMode(@NonNull CarContext context, @NonNull ThemeMode
   @NonNull
   public static ThemeMode getThemeMode(@NonNull CarContext context)
   {
-    final String autoTheme = context.getString(R.string.theme_auto);
     final String lightTheme = context.getString(R.string.theme_default);
     final String nightTheme = context.getString(R.string.theme_night);
-    final String themeMode = getSharedPreferences(context).getString(THEME_KEY, autoTheme);
+    final String followSystemTheme = context.getString(R.string.theme_follow_system);
+    final String themeMode = getSharedPreferences(context).getString(THEME_KEY, followSystemTheme);
 
-    if (themeMode.equals(autoTheme))
-      return ThemeMode.AUTO;
+    if (themeMode.equals(followSystemTheme)) {
+      return ThemeMode.FOLLOW_SYSTEM;
+    }
```

Comment:
One-liners don't need braces.
```suggestion
    if (themeMode.equals(followSystemTheme))
      return ThemeMode.FOLLOW_SYSTEM;
```